src/test/java/hudson/plugins/git/GitToolTest.java

```diff
@@ -43,6 +49,21 @@ public void testForNode() throws Exception {
         assertEquals(gitTool.getGitExe(), newTool.getGitExe());
     }
 
+    @Test
+    public void shouldResolveToolsOnMaster() throws Exception {
+        Assume.assumeTrue(!hudson.remoting.Launcher.isWindows());
```

Comment:
Explanatory comment why this is skipped on Windows?

Reply:
I think I found the reason why it is skipped on Windows.  I've prepared a proposed change to the test so that it can also be run on Windows.